### PR TITLE
Fix fabric startup script issue

### PR DESCRIPTION
**Pull request by Ram-srini**
New minifab by default won't generate the network profiles.
Added command to generate the network profiles.

Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: docker-compose-fabric.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ services:
       - ./examples/apps/generic_client/:/project/avalon/examples/apps/generic_client/
       - ~/.Xauthority:/root/.Xauthority
       - ~/mywork/vars/keyfiles:/keyfiles
-      - ~/mywork/vars/mychannel_network.json:/project/avalon/sdk/avalon_sdk/fabric/network.json
+      - ~/mywork/vars/profiles/mychannel_network_for_pysdk.json:/project/avalon/sdk/avalon_sdk/fabric/network.json
```
Comment (manojgop) [inline docker-compose-fabric.yaml]: How is the file "mychannel_network_for_pysdk.json" getting created ?

File: scripts/start_fabric.sh [hunk 2]
```diff
@@ -83,7 +83,10 @@ fi
 
 # If already fabric network up and running skip start again.
 if [[ ! $(docker ps --format '{{.Names}}' |grep "peer*") ]]; then
+    # Start the fabric containers peers, orderers, cas
     minifab up -i $FABRIC_VERSION
+    # Create network profile file for avalon to use
+    minifab profilegen
```
Comment (manojgop) [inline scripts/start_fabric.sh]: Please add a comment which file is created by execution of this command ?

File: docker-compose-fabric.yaml (hunk 1 above)
Comment (Ram-srini) [reply docker-compose-fabric.yaml]: It is generated from minifab in work_dir(~/mywork/var

File: scripts/start_fabric.sh (hunk 2 above)
Comment (Ram-srini) [reply scripts/start_fabric.sh]: yes

File: docker-compose-fabric.yaml (hunk 1 above)
Comment (manojgop) [reply docker-compose-fabric.yaml]: Please add a comment in docker-compose file and/or startup script

File: scripts/start_fabric.sh (hunk 2 above)
Comment (manojgop) [reply scripts/start_fabric.sh]: Is there an option to pass generated file name with minifab profilegen ? If not, please add a comment which file this command will create

**Pull request merged**